src/renderer.zig

```diff
@@ -166,6 +168,29 @@ pub fn render(playerPosition: Vec3d) void {
 	}
 }
 
+pub fn crosshairDirection(rotationMatrix: Mat4f, fovY: f32, width: u31, height: u31) Vec3f {
+	// stolen code from Frustum.init
+	const invRotationMatrix = rotationMatrix.transpose();
```

Comment:
Surprise: This rotation matrix also "rotates" into OpenGLs coordinate system(y-up), so the inverse actually assumes that you are coming from OpenGLs coordinate system.
Now I think about it, I should probably move that bit into the projection matrix to avoid any further confusion.

Reply:
Fixed in 3d3a1322cb543b5fa16cfe73051ba9910e7fb7d7, please rebase this PR.

Reply:
Done (to my understanding of git)

Reply:
It's not quite what I want, but I guess it's close enough.
What you've done actually looks like a merge rather than a rebase.
What you've done is basically create a new commit with the same changes.
A rebase would literally take the commit and put it into your branch.
You can do this with `git pull --rebase`, it only works if there are no conflicts though.